docs/docs/config/battery.md

```diff
@@ -7,6 +7,15 @@ See the [battery level feature page](../features/battery.md) for more details on
 
 See [Configuration Overview](index.md) for instructions on how to change these settings.
 
+### Kconfig
+
+Definition file: [zmk/app/Kconfig](https://github.com/zmkfirmware/zmk/blob/main/app/Kconfig)
+
+| Config                               | Type | Description                                            | Default |
+| ------------------------------------ | ---- | ------------------------------------------------------ | ------- |
+| `CONFIG_ZMK_BATTERY_REPORTING`       | bool | Enables/disables all battery level detection/reporting | n       |
```

Comment:
While it is technically true that this is disabled by default, it is `imply`d by `ZMK_BLE` so in practice this will typically be true. In order not to confuse users (because they check these pages and think "oh I should enable that") can you add a note below the table that this will be set to true if BLE is enabled?